apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/page.tsx

```diff
@@ -86,18 +128,46 @@ async function ManualEvaluationResults({
   evaluation: EvaluationDto
   commit: Commit
 }) {
+  if (documentLogId) {
+    const targetPage = await findDocumentLogWithEvaluationResultPage({
+      workspaceId: evaluation.workspaceId,
+      documentUuid,
+      commit,
+      documentLogId: documentLogId as string,
+      pageSize: pageSize as string | undefined,
+    }).then((p) => p?.toString())
+
+    if (targetPage && page !== targetPage) {
+      const route = ROUTES.projects
+        .detail({ id: Number(projectId) })
+        .commits.detail({ uuid: commit.uuid })
+        .documents.detail({ uuid: documentUuid })
+        .evaluations.detail(evaluation.id).root
+
+      const targetSearchParams = new URLSearchParams()
+      targetSearchParams.set('page', targetPage)
+      if (pageSize) targetSearchParams.set('pageSize', pageSize as string)
+      targetSearchParams.set('documentLogId', documentLogId as string)
+
+      return redirect(`${route}?${targetSearchParams}`)
```

Comment:
TIL that a class in JS with a `toString` interface is converted to string automatically

Reply:
![image](https://github.com/user-attachments/assets/cd2d851b-4445-4214-9278-f9022a64ad99)